components/Layout.js

```diff
@@ -12,34 +13,6 @@ const CustomLayout = ({ children }) => (
         <Head>
             <title>Apéros Web</title>
             <link rel="icon" href="/favicon.ico" />
-
-            <style>{`
-        .ant-layout-content {
-          padding: 0 2rem !important;
-        }
-
-        .ant-layout-header {
-          height: auto !important;
-          padding: 0 !important;
-        }
-
-        .ant-card .ant-card-body > * ~ * {
-          margin-top: 0.5rem;
-        }
-
-        h2 {
-          margin-top: 1rem !important;
```

Comment:
The `margin-top` is missing

Reply:
No need anymore. The h3 title already have `margin-bottom: 0.5em;`

Reply:
There is a need, the "Communication" title is too close to the "Talks" cards

Reply:
![image](https://user-images.githubusercontent.com/2212144/71558983-2a65ec00-2a59-11ea-82dd-1a355a946fc5.png)